runtime/westend/src/lib.rs

```diff
@@ -364,10 +362,6 @@ parameter_types! {
 	pub const SlashDeferDuration: pallet_staking::EraIndex = 27;
 	pub const RewardCurve: &'static PiecewiseLinear<'static> = &REWARD_CURVE;
 	pub const MaxNominatorRewardedPerValidator: u32 = 64;
-	// quarter of the last session will be for election.
-	pub const ElectionLookahead: BlockNumber = EPOCH_DURATION_IN_BLOCKS / 4;
-	pub const MaxIterations: u32 = 10;
-	pub MinSolutionScoreBump: Perbill = Perbill::from_rational(5u32, 10_000);
```

Comment:
note that `pallet_election_provider_multi_phase` min solution score bump is 1/10_000

Reply:
inconsistency, all should be 5/10_000, thanks